packages/luna/src/locales/ar-eg.json

```diff
@@ -573,6 +576,7 @@
   "labels.invoice.receipt.upload": "إرفاق الإيصال",
   "labels.litespace": "لايت سبيس",
   "labels.delete": "حذف",
+  "labels.coming-soon": "قريبا 🎉",
```

Comment:
It seems that there are more than one feature that will come in the next versions so I think it will be better to leave this id in the labels instead writing it many times in different places.